questionnaire/form-helper.js

```diff
@@ -44,7 +54,7 @@ function renderSection({
     cspNonce
 }) {
     const showButton = !isFinal;
-    const isSummary = checkIsSummary(sectionId);
+    const isSubmission = checkIsSubmission(sectionId);
```

Comment:
For readability change these to ` const isSubmissionContext`, and `checkIsSubmissionContext()`, or `isSubmissionContext()` to align it with the UISchema terminology. I think saying "is submission" has a different meaning to "is submission context/page" because a "submission" is a resource